pkg/runtime/masters.go

```diff
@@ -50,13 +51,13 @@ const (
 	RemoteReplaceKubeConfig     = `grep -qF "apiserver.cluster.local" %s  && sed -i 's/apiserver.cluster.local/%s/' %s && sed -i 's/apiserver.cluster.local/%s/' %s`
 	RemoteJoinMasterConfig      = `echo "%s" > %s/etc/kubeadm.yml`
 	InitMaster115Lower          = `kubeadm init --config=%s/etc/kubeadm.yml --experimental-upload-certs`
-	JoinMaster115Lower          = "kubeadm join %s:6443 --token %s --discovery-token-ca-cert-hash %s --experimental-control-plane --certificate-key %s"
-	JoinNode115Lower            = "kubeadm join %s:6443 --token %s --discovery-token-ca-cert-hash %s"
+	JoinMaster115Lower          = "kubeadm join %s --token %s --discovery-token-ca-cert-hash %s --experimental-control-plane --certificate-key %s"
+	JoinNode115Lower            = "kubeadm join %s --token %s --discovery-token-ca-cert-hash %s"
 	InitMaser115Upper           = `kubeadm init --config=%s/etc/kubeadm.yml --upload-certs`
 	JoinMaster115Upper          = "kubeadm join --config=%s/etc/kubeadm.yml"
 	JoinNode115Upper            = "kubeadm join --config=%s/etc/kubeadm.yml"
 	RemoveKubeConfig            = "rm -rf /usr/bin/kube* && rm -rf ~/.kube/"
-	RemoteCleanMasterOrNode     = `if which kubeadm;then kubeadm reset -f %s;fi && \
+	RemoteCleanMasterOrNode     = `systemctl restart docker kubelet;if which kubeadm;then kubeadm reset -f %s;fi && \
```

Comment:
Why does the delete operation have to restart the service instead of stopping the service?